plugins/hub/osbuild.py

```diff
@@ -246,4 +250,10 @@ def osbuildImage(name, version, distro, image_type, target, arches, opts=None, p
     if priority and priority < 0 and not context.session.hasPerm('admin'):
         raise koji.ActionNotAllowed('only admins may create high-priority tasks')
 
-    return kojihub.make_task('osbuildImage', args, **task)
+    # If task_id is returned from Koji Hub we assume
+    # that the task has been added to the database
+    task_id = kojihub.make_task('osbuildImage', args, **task)
+    if task_id:
```

Comment:
Doesn't hurt I guess, but I don't actually think this can be anything but a positive integer, see https://pagure.io/koji/blob/master/f/hub/kojihub.py#_738

```
CREATE TABLE task (
    id SERIAL NOT NULL PRIMARY KEY,
```

see also [`SERIAL`](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-SERIAL)

Reply:
For completeness from the [docs](https://www.postgresql.org/docs/current/functions-sequence.html): `If the sequence object has been created with default parameters, successive nextval calls will return successive values beginning with 1.`

Reply:
From the source I think `kojihub.make_task('osbuildImage', args, **task)` will never return `None` or `0`. Either you have a successful call and a `task_id > 0` or an exception. It is not a big deal though.

Reply:
Looks like @gicmo intended this to be merged, so I'll resolve and let the magic happen ;)